index.bs

```diff
@@ -989,13 +993,29 @@ remote-playback-modify-request message with the following values:
 : controls
 :: Updated controls as defined in {#remote-playback-state-and-controls}
 
+: remoting-audio
+:: Audio encodings that the controller can send to the receiver once they
+    are acked via the remote-playback-modify-response message.
+
+: remoting-video
+:: Videos encodings that the controller can send to the receiver once they
+    are acked via the remote-playback-modify-response message.
+
+: remoting-stats-interval
```

Comment:
Should this go in the remote-playback-start-request and remote-playback-start-response?
That is where the intervals are determined for a regular streaming session.

Reply:
I moved it there.